### ISSUE TITLE
NullPointerException if opening address book without any addresses

### ISSUE THREAD

**Issue by Heinerion** [Heinerion/InvoiceManager#34]
Stacktrace:
```haskell
Exception in thread "Thread-0" java.lang.NullPointerException
	at java.base/java.util.Objects.requireNonNull(Objects.java:222)
	at de.heinerion.betriebe.data.MemoryBank.getSortedEntries(MemoryBank.java:121)
	at de.heinerion.betriebe.data.MemoryBank.getAddresses(MemoryBank.java:41)
	at de.heinerion.betriebe.data.DataBase.getAddresses(DataBase.java:60)
	at de.heinerion.invoice.view.swing.home.receiver.AddressChooserPanel.refreshBoxes(AddressChooserPanel.java:139)
	at de.heinerion.invoice.view.swing.home.receiver.AddressPanel.refresh(AddressPanel.java:21)
	at de.heinerion.invoice.view.swing.home.ReceiverPanel.refresh(ReceiverPanel.java:54)
	at de.heinerion.invoice.view.swing.home.ApplicationFrameImpl.refreshBoxes(ApplicationFrameImpl.java:127)
	at de.heinerion.invoice.view.swing.home.ApplicationFrameImpl.refresh(ApplicationFrameImpl.java:119)
	at de.heinerion.invoice.view.GuiStarter.collectData(GuiStarter.java:41)
	at de.heinerion.invoice.view.GuiStarter.lambda$startDataThread$0(GuiStarter.java:33)
	at java.base/java.lang.Thread.run(Thread.java:832)
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
	at java.base/java.util.Objects.requireNonNull(Objects.java:222)
	at de.heinerion.betriebe.data.MemoryBank.getSortedEntries(MemoryBank.java:121)
	at de.heinerion.betriebe.data.MemoryBank.getAddresses(MemoryBank.java:41)
	at de.heinerion.betriebe.data.DataBase.getAddresses(DataBase.java:60)
	at de.heinerion.invoice.view.swing.menu.AddressBookMenuEntry.createWidgets(AddressBookMenuEntry.java:29)
	at de.heinerion.invoice.view.swing.menu.MenuEntry.showDialog(MenuEntry.java:39)
	at de.heinerion.invoice.view.swing.menu.MenuEntry.showDialog(MenuEntry.java:33)
	at de.heinerion.invoice.view.swing.menu.MenuBar.lambda$createItem$0(MenuBar.java:36)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.AbstractButton.doClick(AbstractButton.java:369)
	at java.desktop/javax.swing.plaf.basic.BasicMenuItemUI.doClick(BasicMenuItemUI.java:1020)
	at java.desktop/javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(BasicMenuItemUI.java:1064)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6636)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
	at java.desktop/java.awt.Component.processEvent(Component.java:6401)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5012)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4844)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4919)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4548)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4489)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2764)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4844)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by Heinerion** [comment]
can't even get back to the application

**Comment by Heinerion** [comment]
fixed. Can't really tell, which commit it has been, but it should have been fixed around 63a1ef4bb9c6570f00adb422e71ec2a190252cf8

The responsible class has been deleted since a while now
